crates/fluvio-cli/src/client/topic/create.rs

```diff
@@ -258,6 +270,21 @@ fn validate(name: &str, _spec: &TopicSpec) -> Result<()> {
     Ok(())
 }
 
+fn create_deduplication(dedup_count: u64, dedup_age: Option<Duration>) -> Deduplication {
```

Comment:
this should be builder pattern so can customize this

Reply:
I noticed that Deduplication, Bounds and Filter implements Builder, but seems a little verbose to create 3 Builders for this, no?

Maybe with additional methods at Deduplication?